packs/chatops_tests/actions/test_aliases_with_slack.py

```diff
@@ -144,12 +144,18 @@ def test_help_shortcut(self):
             if filtered_messages:
                 messages.extend(filtered_messages)
 
-        self.assertEqual(1, len(messages))
-        if len(messages) != 1:
+        self.assertEqual(2, len(messages))
+        if len(messages) != 2:
             time.sleep(self.WAIT_FOR_MESSAGES_TIMEOUT)
 
+        # Help commands should returns more than 100
+        commands_len = 0
+        for message in messages:
+            commands_len = commands_len + len(message['text'].split('\n'))
+        self.assertGreater(commands_len, 100)
+
         # Help commands don't get acked
-        self.assertIn("!help - Displays all of the help commands that this bot knows about.", messages[0]['text'])
+        self.assertIn("!help - Displays all of the help commands that this bot knows about.", messages[1]['text'])
```

Comment:
Once you have the `combined_text` from above, you won't need to search a specific message for this. That will make this test less brittle.